libbpf-tools/ksnoop.c

```diff
@@ -707,8 +707,8 @@ static void trace_handler(void *ctx, int cpu, void *data, __u32 size)
 			size, sizeof(trace) - MAX_TRACE_BUF);
 		return;
 	}
-	printf("%16lld %4d %8u %s(\n", trace->time, trace->cpu, trace->pid,
-	       trace->func.name);
+	printf("%16lld %4d %8u %8u %16s %32s(\n", trace->time, trace->cpu,
+	       trace->pid_ns_id, trace->pid, trace->comm, trace->func.name);
```

Comment:
Can you add an new option like `verbose` to this tool, and when enabled, print these extra fields.

This would keep the default output succinct.

Reply:
OK, I added the verbose option.